### DISCUSSION TITLE
Should be usefull the Wiki section?

### DISCUSSION THREAD

**Discussion by unix4you2** [adminerevo/adminerevo#19] [Ideas]
Hi!
Do you think that enables the Wiki in this repo should be usefull to do some documentation?  I think that this could be a space that the project could use to have an entry point as web / docs. 
Regards

**Comment by LionelLaffineur** [comment]
Dear,
Someone is already working on documentation.  I don't know yet the details on how it will be structured.
Keep looking for an announcement.
Regards
